drivers/include/net/netdev.h

```diff
@@ -54,6 +54,7 @@ extern "C" {
 #include <sys/uio.h>
 
 #include "net/netstats.h"
+#include "net/l2filter.h"
```

Comment:
*If* a clash occurs with a header from *our* codebase the header is flawed IMHO, not the consumer of that header (in the same codebase). I would rather take a look into the header and assure the clash does not happen there (e.g. with proper prefixing according to our naming conventions), and not in the consumers of that header. This is how you build a proper API: not putting the need to remember exceptions on the shoulders of the "user". Mind you, that is not true for external headers, but there you still have can name clashes in the linker (see emb6's clist e.g.) but those need to be fixed by other means anyway.